pid_velocity_right:
  ros__parameters:
    encoder_count_minimum: -2147483648
    encoder_count_maximum: 2147483647
    encoder_counts_per_rev: 2248.86
    wheel_diameter_meters: 0.04186
    wheel_separation_meters: 0.1875
    motor_power_minimum: -400
    motor_power_maximum: 400
    motor_power_maximum_step: 25
    velocity_threshold: 0.004
    num_velocity_samples: 7
    kp: 0.35
    ki: 0.016
    kd: 0.008
    pid_frequency_hz: 8
    pid_integral_windup_limit: 1.0
    pid_output_limit: 0.1
    motor_velocity_power_factor: 0.000245

pid_velocity_left:
  ros__parameters:
    encoder_count_minimum: -2147483648
    encoder_count_maximum: 2147483647
    encoder_counts_per_rev: 2248.86
    wheel_diameter_meters: 0.04186
    wheel_separation_meters: 0.1875
    motor_power_minimum: -400
    motor_power_maximum: 400
    motor_power_maximum_step: 25
    velocity_threshold: 0.004
    num_velocity_samples: 7
    kp: 0.35
    ki: 0.016
    kd: 0.008
    pid_frequency_hz: 8
    pid_integral_windup_limit: 1.0
    pid_output_limit: 0.1
    motor_velocity_power_factor: 0.000245
